.devcontainer/devcontainer.json

```diff
@@ -2,12 +2,12 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/rust
 {
 	"name": "Rust",
-	"image": "ghcr.io/dojoengine/dojo-dev:72f7bc2220f0438e435052f807e58c09b7b61cbe",
+	"image": "mcr.microsoft.com/devcontainers/rust:dev-bookworm",
```

Comment:
I would prefer to use the `.devcontainer/Dockerfile` as a base image, which gets built in the CI

We can update that variant here: https://github.com/dojoengine/dojo/blob/d7e9788459f3b2e797b6565f1057c96a6f5f8cc2/.github/workflows/devcontainer.yml#L43

We should also remove the old devenet here: https://github.com/dojoengine/dojo/blob/d7e9788459f3b2e797b6565f1057c96a6f5f8cc2/.devcontainer/Dockerfile#L31

And update it to install Bun cc @ponderingdemocritus 

Then update this git hash once it is built